cmd/diff-main.go

```diff
@@ -67,11 +73,33 @@ type diffMessage struct {
 	FirstURL      string       `json:"first"`
 	SecondURL     string       `json:"second"`
 	Diff          differType   `json:"diff"`
+	FirstSize     int64        `json:"firstSize"`
+	SecondSize    int64        `json:"secondSize"`
 	Error         *probe.Error `json:"error,omitempty"`
 	firstContent  *clientContent
 	secondContent *clientContent
 }
 
+type diffTotalSize struct {
+	Status     string `json:"status"`
+	SourceSize int64  `json:"sourceSize"`
+	TargetSize int64  `json:"targetSize"`
+}
+
+func (t diffTotalSize) String() string {
+	msg := console.Colorize("DiffSize", fmt.Sprintf("SourceSize: %s TargetSize: %s",
```

Comment:
Probably we should only print the difference between sizes, easier for the user to read.

LGTM otherwise.